src/hub/Hub.sol

```diff
@@ -262,6 +269,41 @@ contract Hub is Circles {
         emit InviteHuman(msg.sender, _human);
     }
 
+    /**
+     * Invite human as organization allows to register a human avatar as an organization.
+     * @param _human address of the human to invite
+     * @param _donationReceiver address of where to send the donation to with 2300 gas (using transfer)
+     */
+    function inviteHumanAsOrganization(address _human, address payable _donationReceiver) external payable {
+        require(msg.value > MINIMUM_DONATION, "Donation must be at least 0.1 xDai.");
+        require(isOrganization(msg.sender), "Only organizations can invite.");
+
+        // insert avatar into linked list; reverts if it already exists
+        _insertAvatar(_human);
+
+        // set the last mint time to the current timestamp for invited human
+        // and register the v1 Circles contract status
+        address v1CirclesStatus = _avatarV1CirclesStatus(_human);
```

Comment:
Let's refactor the common parts of `inviteHuman` and `inviteHumanAsOrganization` into an _inviteHuman function

Reply:
yes, I wanted to, but then thought I was overthinking it. Ill quickly do

Reply:
turned out it was well worth the refactoring!